wtxmgr/migrations.go

```diff
@@ -36,3 +40,63 @@ func getMigrationsToApply(version uint32) []dbVersion {
 	}
 	return migrations
 }
+
+// migrationBucketMinedInputs is the migration responsible for creating a new
```

Comment:
As this is a migration touches a sensitive section of the wallet (the tx tracking), we should add a test for the migration itself. This test should ensure that it can handle: upgrade from zero, upgrade to same version, upgrade w/ a higher db version (as we hit version 5 before this). 

For inspiration, check out the way we test migrations within lnd. https://github.com/lightningnetwork/lnd/blob/master/channeldb/migrations_test.go